config/dev.cfg

```diff
@@ -16,7 +16,8 @@ credential.password=testPwd
 
 [path]
 path.ebooks=ebooks
-path.extras=ebooks/extras
+path.extras=extras
+path.separate=true
```

Comment:
The first package of improvements: 5015a87
Config now do not even have to contain path.ebooks and path.extras. It can be the same path.separate.ebooks and path.separate.extras.